steps/cadence-innovus-flowsetup/setup.tcl

```diff
@@ -77,6 +77,7 @@ set vars(library_sets)        "libs_typical"
 
 set vars(libs_typical,timing) [join "
                                 $vars(adk_dir)/stdcells.lib
+                                $vars(adk_dir)/stdcells-pm.lib
                                 [glob -nocomplain $vars(adk_dir)/iocells.lib]
```

Comment:
Can you wrap the stdcells-pm.lib with a "glob -nocomplain" like was done here?

The glob is basically doing a wildcard *. With the glob, it will not error out if it does not find stdcells-pm.lib.

Reply:
It's updated.